doc/model.rst

```diff
@@ -0,0 +1,206 @@
+Standard (``nonDimensional=1``)                                        Legacy PlasComCM (``nonDimensional=2``)
+====================================================================== =============================================================================
+:math:`u^*_\infty = \sqrt \frac{P^*_\infty}{\rho^*_\infty}`            :math:`u^*_\infty = \sqrt \frac{\gamma P^*_\infty}{\rho^*_\infty}`
+:math:`e^*_\infty = (u^*_\infty)^2 = \frac{P^*_\infty}{\rho^*_\infty}` :math:`e^*_\infty = (u^*_\infty)^2 = \frac{\gamma P^*_\infty}{\rho^*_\infty}`
+:math:`\rho = \rho^* /\rho^*_\infty`                                   :math:`\rho = \rho^* /\rho^*_\infty`
+:math:`P = P^* /P^*_\infty`                                            :math:`P = P^* /(\rho^*_\infty (u^*_\infty)^2)`
+:math:`T = T^* /T^*_\infty`                                            :math:`T = T^* /((\gamma-1)T^*_\infty)`
+:math:`u_i = u^*_i /u^*_\infty`                                        :math:`u_i = u^*_i /u^*_\infty`
+:math:`e = e^* /e^*_\infty`                                            :math:`e = e^* /e^*_\infty`
+:math:`t = t^* /(L^* / u^*_\infty)`                                    :math:`t = t^* /(L^* / u^*_\infty)`
+:math:`x_i = x_i^* /L^*`                                               :math:`x_i = x_i^* /L^*`
+====================================================================== =============================================================================
+
+Substitution into the dimensional form of the Navier-Stokes equations yields
+the non-dimensional equivalent
+
+.. math::
+    \frac{\partial \rho}{\partial t} + \frac{\partial }{\partial x_j} \rho u_j &=
+       S_\rho \\
+    \frac{\partial \rho u_i}{\partial t} + \frac{\partial}{\partial x_j}\left(\rho u_i u_j
+       + p\delta_{ij} - \tau_{ij}\right) &= S_{\rho u_i} \\
+    \frac{\partial \rho E}{\partial t} +
+      \frac{\partial}{\partial x_j}\left(\left\{\rho E + p\right\}u_j +
+      q_j - u_i \tau_{ij}\right) &= S_{\rho E}
+
+with the following non-dimensionalization for the source terms
+
+.. math::
+    S_\rho        &= \frac{S^*_\rho L^*}{\rho^*_\infty U^*_\infty} \\
+    S_{\rho u_i}  &= \frac{S^*_{\rho u_i } L^*}{\rho^*_\infty (U^*_\infty)^2 } \\
+    S_{\rho E}    &= \frac{S^*_{\rho E} L^*}{\rho^*_\infty (U^*_\infty)^3}
+
+by choosing the following non-dimensionalizations for the transport coefficients
+
+.. math::
+    \mu       &= \mu^* /\mu^*_\infty \\
+    \lambda   &= \lambda^* /\lambda^*_\infty \\
+    \kappa   &= \kappa^* /\kappa^*_\infty \\
+
+the non-dimensional viscous stress tensor and heat flux vector can be written as
+
+.. math::
+    \tau_{ij} &= \frac{\mu}{\RE} \left(\frac{\partial u_i}{\partial x_j} +
+      \frac{\partial u_j}{\partial x_i}\right) +
+      \frac{\lambda}{\RE} \frac{\partial u_k}{\partial x_k}\delta_{ij} \\
+    q_i &= - \frac{\mu}{\RE \Pr} \frac{\partial T}{\partial x_i}
+
+where :math:`\RE` is defined as the code Reynolds number,
```

Comment:
`\RE` fails in `latexpdf`.  How can we add a `newcommand` in rst?

Reply:
6b92ef4